docs/oracle/oracle-server.md

```diff
@@ -56,6 +56,12 @@ For more information on how to use our built in `cli` to interact with the serve
   - `label` - Label for this event
   - `maturationtime` - The earliest expected time an outcome will be signed, given in epoch second
   - `outcomes` - Possible outcomes for this event
+- `createrangedevent` `name` `maturationtime` `start` `stop` `step` - Registers an oracle event with a range of outcomes
```

Comment:
Let's do code samples fo this as well for both creation and signing to be explicit.